Marlin/language_de.h

```diff
@@ -153,6 +159,12 @@
 #define MSG_ERR_MINTEMP                     LCD_STR_THERMOMETER " UNTERSCHRITTEN"
 #define MSG_ERR_MAXTEMP_BED                 "BETT " LCD_STR_THERMOMETER " ÜBERSCHRITTEN"
 #define MSG_ERR_MINTEMP_BED                 "BETT " LCD_STR_THERMOMETER " UNTERSCHRITTEN"
+#define MSG_ERR_Z_HOMING                    "G28 Z Forbidden"
+#define MSG_HALTED                          "DRUCKER STOPP"
+#define MSG_PLEASE_RESET                    "Bitte Resetten"
+#define MSG_SHORT_DAY                       "t" // One character only
+#define MSG_SHORT_HOUR                      "h" // One character only
+#define MSG_SHORT_MINUTE                    "m" // One character only
 #define MSG_HEATING                         "Extr. Aufheizen..."
```

Comment:
Since this is a message indicating that the extruder is currently heating "Extr. heizt..." makes more sense here.
"Extr. Aufheizen" would fit better as a menu message which you select in order to heat up the extruder.